<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>888</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Measurement Log</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Experiment.qrc">
    <normaloff>:/petersIcons/icons/Owwnotebook_icon.png</normaloff>:/petersIcons/icons/Owwnotebook_icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_12">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="timespanComboBox"/>
     </item>
     <item>
      <widget class="QToolButton" name="refreshButton">
       <property name="toolTip">
        <string>Repeat database Query</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Experiment.qrc">
         <normaloff>:/openicon/icons/view-refresh-7.png</normaloff>:/openicon/icons/view-refresh-7.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fromLabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>From:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="fromDateTimeEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="toLabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>To:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="toDateTimeEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitterHorizontal">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSplitter" name="splitterLeftVertical">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTableView" name="studyTableView"/>
         <widget class="QWidget" name="layoutWidget_2">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QToolButton" name="scanNameFilterButton">
               <property name="toolTip">
                <string>Apply filter</string>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="Experiment.qrc">
                 <normaloff>:/kde/icons/view-filter.png</normaloff>:/kde/icons/view-filter.png</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="scanNameTableView"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Plot</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="4" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>y-axis</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QComboBox" name="xComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="9" column="1" colspan="2">
           <widget class="QComboBox" name="windowComboBox"/>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QComboBox" name="yComboBox"/>
          </item>
          <item row="12" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="plotButton">
              <property name="text">
               <string>Plot</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="updatePlotButton">
              <property name="toolTip">
               <string>Update current</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="Experiment.qrc">
                <normaloff>:/openicon/icons/view-refresh-7.png</normaloff>:/openicon/icons/view-refresh-7.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="updateAllPlotsButton">
              <property name="toolTip">
               <string>Update all</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="Experiment.qrc">
                <normaloff>:/openicon/icons/view-refresh-7.png</normaloff>:/openicon/icons/view-refresh-7.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>window</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>x-axis</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>x-unit</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QLineEdit" name="xUnitEdit"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>y-unit</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QLineEdit" name="yUnitEdit"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitterVertical">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableView" name="measurementTableView"/>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Long comment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitterHorizontalParam">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Parameters</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="parameterTableView"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Results</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="resultTableView"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>gui.MeasurementLogUi.PlainTextEdit</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Experiment.qrc"/>
 </resources>
 <connections/>
</ui>
